README.md

```diff
@@ -1,2 +1,27 @@
 # comp110-worksheet-B
```

Comment:
Flowchart is clear and sensible, however it seems to be describing the player's thought process rather than the implementation of the game logic. Be sure to read specifications carefully -- this is an important skill for software developers!